chan-plugin/chan-plugin.php

```diff
@@ -0,0 +1,50 @@
+<?php /**
+ * Plugin Name: chan-plugin
+ * Plugin URI: ''
+ * Author: Chandan
+ * Author URI: ''
+ * Version: 2.3.5
+ * Description: 'A new Plugin'
+ * Text Domain: 'xyz'
+ * *
+ * 
+ */
+
+ //Function to handle a shortcode
+
+function handle_shortcode( $attr, $content='' ){
+    global $post;       //global post variable
+
+    $attr = shortcode_atts( array(      //default attribute values
+        'title' => 'Default title',
```

Comment:
All hardcoded strings should be transalatable
https://codex.wordpress.org/I18n_for_WordPress_Developers

Reply:
Translation logic has been added